client/js/components/header/search.js

```diff
@@ -0,0 +1,44 @@
+import showHide from './../show-hide';
+import { KEYS } from './../../util';
+
+const headerSearch = (el) => {
+  const headerSearchForm = el.querySelector('.js-search');
+  const headerSearchInput = headerSearchForm.querySelector('.js-search-input');
```

Comment:
We could do this by using the `form.properties` to stop needing to use classnames for selectors... seems a little more semantic to what we're trying to do.